### PR TITLE
UI enhancement

### LINKED ISSUE #850: UI elements overlap when referenced entity has long organisation and/or repository names
## Expected Behavior

Organisation & repo name text is truncated or filter area is scrollable.

## Current Behavior

Organisation & repo name text is cover with table's filter area.

Screenshot 2023-02-13 at 14 32 53

## Steps to Reproduce

Follow installation steps in the plugin README. Create repository with long name for adding it to the entity catalog or manually edit HTML in the dev tools to contain longer test.

## Possible Solution

Truncate logs table header text or add possibility to scroll filter area when repository name exceeds maximum length.

screenshot1

screenshot2

## Your Environment

- `@roadiehq/backstage-plugin-github-pull-requests` version: **2.5.0**

### PR DESCRIPTION
**Pull request by frenchbread**
closes #850 

#### :heavy_check_mark: Checklist

- [ ] Added tests for new functionality and regression tests for bug fixes
- [x] Added changeset (run `yarn changeset` in the root)
- [x] Screenshots of before and after attached (for UI changes)
- [ ] Added or updated documentation (if applicable)

### changes

> Fixes bug in PullRequestsTableView component where table layout did not readjust properly when table header text length exceeded.

### screenshot before

screenshot1

### screenshot after

screenshot2
screenshot3

### CODE REVIEW

File: plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsTable/PullRequestsTable.tsx [hunk 1]
```diff
@@ -137,6 +137,17 @@ export const PullRequestsTableView: FC<Props> = ({
...
+    const elem = document.querySelector('.gh-pull-requests-table-header');
+
+    const targetElement = elem?.parentElement?.parentElement?.parentElement;
+
+    if (targetElement) {
+      targetElement.style.display = 'block';
+      targetElement.style.paddingRight = '24px';
+    }
+  }, []);
+
```
Comment (kissmikijr) [inline plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsTable/PullRequestsTable.tsx:150]: I do not really like this solution. Can you please try to do this with CSS only? I think something like setting the overflow on the title element would bring us 90% there.

My other thought is that this <Table/> component is coming from backstage/backstage. I think we should try to fix this title in the base component's handling of the title prop so every backstage table would be able to handle the overflow of big titles

Comment (frenchbread) [reply plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsTable/PullRequestsTable.tsx]: @kissmikijr Thanks for your comment. I've updated PR to make is CSS only. Result looks almost the same.

**Comment by kissmikijr** [comment]
We'll need one more rebase, the build pipeline was broken and the fix should be out now in the main branch

**Review by kissmikijr — APPROVED** [review]

**Comment by kissmikijr** [comment]
Thank you, let's merge it!

**Pull request merged**